imports/lib/models/puzzles.js

```diff
@@ -1,14 +1,33 @@
 import { _ } from 'meteor/underscore';
+import Mustache from 'mustache';
 import { huntsMatchingCurrentUser } from '../../model-helpers.js';
 import Base from './base.js';
 import PuzzlesSchema from '../schemas/puzzles.js';
 import ActiveOperatorRole from '../active-operator-role.js';
 
-const Puzzles = new Base('puzzles', {
-  transform(doc) {
-    return _.extend({}, doc, { tags: _.uniq(doc.tags) });
-  },
-});
+// Disable escaping - we're passing this in as an attribute, not rendering it as
+// HTML, so we don't need to escape. Pretty not great that this is only
+// available as a global setting.
+Mustache.escape = s => s;
```

Comment:
Instead of quietly disabling escaping everywhere, can we just make the operator responsible for using triple-braces or the `&variable` syntax instead of double-braces to indicate that they don't want things to be escaped?  This seems like the sort of thing that will happen once per hunt and which you or I can be responsible for getting right